tests/dialects/test_dialect.py

```diff
@@ -1444,6 +1444,36 @@ def test_cross_join(self):
             },
         )
 
+        # UNNEST without column alias
+        self.validate_all(
+            "SELECT * FROM x CROSS JOIN UNNEST(y) AS t",
+            write={
+                "presto": "SELECT * FROM x CROSS JOIN UNNEST(y) AS t",
+                "spark": "SELECT * FROM x LATERAL VIEW EXPLODE(y) t AS t_struct",
+                "databricks": "SELECT * FROM x LATERAL VIEW EXPLODE(y) t AS t_struct",
+            },
+        )
+
+        # UNNEST STRUCT Object into multiple columns, using single alias
+        self.validate_all(
+            "SELECT a, b FROM x CROSS JOIN UNNEST(y) AS t (a, b)",
+            write={
+                "presto": "SELECT a, b FROM x CROSS JOIN UNNEST(y) AS t(a, b)",
+                "spark": "SELECT a, b FROM x LATERAL VIEW EXPLODE(y) t AS t_struct",
+                "hive": "SELECT a, b FROM x LATERAL VIEW EXPLODE(y) t AS t_struct",
```

Comment:
This doesn't look correct, was it tested on Spark? We've changed the column aliases from `(a, b)` to `t_struct`, we shouldn't be able to select them right? What's the behavior of exploding multiple columns and having only 1 alias?

Reply:
Okay, this is strange, and I just noticed!!
I tested the optimized query which works fine on Spark, in which it was able to reference the columns correctly from `t_struct`
please check ref: I'll cross check and see if I can add the reference back above in tranpiled itself.
<img width="608" alt="Screenshot 2024-11-04 at 6 05 15 PM" src="https://github.com/user-attachments/assets/da00655e-842b-4d76-a20d-e4a19a0adf56">

Reply:
Reverted `t_struct` change here,

Transpiled and optimized spark will look like this(No Change): Presto to Spark

- Presto Query 
```
SELECT a, b,c,d FROM x CROSS JOIN UNNEST(y) AS t(a, b)
```
- Spark Transpile
```
SELECT a, b, c, d FROM x LATERAL VIEW EXPLODE(y) t AS a, b
```
- Spark Optimize
```
SELECT
  `t`.`a` AS `a`,
  `t`.`b` AS `b`,
  `x`.`c` AS `c`,
  `x`.`d` AS `d`
FROM `x` AS `x`
LATERAL VIEW
EXPLODE(`x`.`y`) t AS `a`, `b`
```